src/Integration.UnitTests/Telemetry/TelemetryHelper_CreatePayload.cs

```diff
@@ -34,13 +35,15 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     [TestClass]
     public class TelemetryHelper_CreatePayload
     {
+        private static readonly BindingConfiguration ValidBindingConfiguration = CreateConfiguration(SonarLintMode.Connected, "https://sonarcloud.io");
```

Comment:
Miscellaneous - added symbolic value to make reduce duplication and make the tests easier to read